vdev/src/commands/check/licenses.rs

```diff
@@ -9,6 +9,9 @@ pub struct Cli {}
 
 impl Cli {
     pub fn exec(self) -> Result<()> {
-        app::exec("rust-license-tool", ["check"], true)
+        app::exec("rust-license-tool", ["check"], true).map_err(|err| {
+            eprintln!("Run `cargo vdev build licenses` to regenerate the file");
```

Comment:
Probably should use the `warn!` or `error!` macros here to handle this output.

Reply:
👍 I used `info!` since it seemed like that fit best for this message.